test/unit/foreman/renderer/scope/macros/snippet_rendering_test.rb

```diff
@@ -40,7 +40,7 @@ class SnippetRenderingTest < ActiveSupport::TestCase
   test "should render a snippets with variables" do
     Foreman::Deprecation.expects(:deprecation_warning).once
```

Comment:
This will probably need to be removed for the test to pass.

Reply:
thanks!